src/bundle/Resources/config/default_settings.yml

```diff
@@ -0,0 +1,28 @@
+parameters:
+    ez_recommendation.default.recommendation.customer_id: ~
+    ez_recommendation.default.recommendation.license_key: ~
+    ez_recommendation.default.server_uri: ~
+    ez_recommendation.default.recommendation.included_content_types: []
+    ez_recommendation.default.author_id: 14
+    ez_recommendation.response.random_content_types: []
```

Comment:
I think you should document somewhere, that ContentTypes used for random responses *have* to have a field with identifier `intro`. Otherwise, it might lead to `Notice: Trying to get property 'xml' of non-object` in `RecommendationRandomContentEventSubscriber::getRandomRecommendationItems`.